app/uk/gov/hmrc/tariffclassificationfrontend/connector/ConnectorHttpClient.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.connector
+
+import com.typesafe.config.Config
+import javax.inject.Inject
+import play.api.libs.ws.WSClient
+import uk.gov.hmrc.http._
+import uk.gov.hmrc.http.hooks.HttpHook
+import uk.gov.hmrc.play.http.ws._
+
+class ConnectorHttpClient @Inject()(config: Option[Config], client: WSClient) extends HttpGet with WSGet
```

Comment:
We need to use `http-verbs`.
I might be wrong, but I think you are not using it here.
See this example: 
https://github.com/hmrc/api-revocation-frontend/blob/master/app/config/frontendWiring.scala
https://github.com/hmrc/api-revocation-frontend/blob/master/app/connectors/DelegatedAuthorityConnector.scala

Reply:
It uses http-verbs underneath but I'll change it anyway